pallets/cash/src/internal/initialize.rs

```diff
@@ -115,7 +116,7 @@ pub fn initialize_block<T: Config>(now: Timestamp) -> Result<(), Reason> {
     }
 
     <Module<T>>::deposit_event(Event::TransferCash(
-        ChainAccount::Reserved,
+        GATEWAY_VOID,
```

Comment:
Not sure what this is?

Reply:
This is where we transfer from to miners...and now we have `ChainAccount::Gate(...)` instead of `ChainAccount::Reserved`, so its just shorthand for `ChainAccount::Gate([0u8; 32])`